config/manifests/bases/l5-operator.clusterserviceversion.yaml

```diff
@@ -5,7 +5,8 @@ metadata:
     alm-examples: '[]'
     capabilities: Full Lifecycle
     certified: "false"
-    description: L5 Operator is a demo operator that aims to exhibits level 5 capabilities for web applications.
+    description: L5 Operator is a demo operator that aims to demonstrate level 5 capabilities
+      for web applications.
```

Comment:
For web applications or stateful applications? What do you think?

Reply:
ours is a stateless webapp .. thought the description should be neutral though..